packages/main/src/themes/Token.css

```diff
@@ -22,10 +22,20 @@
 	box-shadow: var(--sapContent_Interaction_Shadow);
 }
 
-:host([selected]:not([readonly]):not([focused])) {
+:host([selected]:not([readonly])) {
 	color: var(--sapButton_Selected_TextColor);
+}
+
+:host([selected]:not([readonly]):not([focused])) {
 	background: var(--sapButton_Selected_Background);
 	border: var(--sapButton_BorderWidth) solid var(--sapButton_Selected_BorderColor);
+	box-shadow: var(--_ui5_token_selected_box_shadow);
```

Comment:
Is the valid only for Morning and Evening Horizon, as it is not specified in the specification but currently not applied to the HC themes. I noticed that on the images attached in the specification for HC themes, there is no box-shadow applied, but that is not source of truth as it is not mentioned explicitly.

Reply:
Something more, the border and box-shadow applied result in different visualisation from the one shown in the specification. Did you ask visual designers for review from their side in order to approve that the ui5-token looks as expected? If so, please ask them to adjust the screenshot provided in the specification.